AmericanMedicalAssociation/ama-style-guide-2
# Adf 129 prototype search results

## Ticket(s)
**Do not** submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Github Issue**
- [Issue 129: Prototype search results](https://palantir.atlassian.net/browse/ADF-129)

**Jira Ticket**
- [ADF-129: Prototype search results](https://palantir.atlassian.net/browse/ADF-129)

## Description
Show different “states” – Member (member badge) / Non-member (no member badge) / Verified (verified check-mark) / Not-verified (no check-mark)


## To Test
Navigate to the Dr-finder search results header and you will see the following 
<img width="364" alt="Screen Shot 2022-10-06 at 11 27 08 AM" src="https://user-images.githubusercontent.com/89882376/194367682-c3d8f7e0-3bea-4d07-990b-8ebfa1928b1e.png">


## Relevant Screenshots/GIFs
Use something like [Skitch](https://evernote.com/skitch/) or [GIPHY Capture](https://giphy.com/apps/giphycapture) to capture images/gifs to demonstrate behaviors.


## Remaining Tasks
Create if statements so that the member box will be displayed 
Create if statement for the check mark verified

## Additional Notes

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
